api/BUILD

```diff
@@ -19,84 +19,54 @@ proto_library(
         "//envoy/api/v2/ratelimit:pkg",
         "//envoy/api/v2/route:pkg",
         "//envoy/api/v3alpha:pkg",
-        "//envoy/api/v3alpha/auth:pkg",
-        "//envoy/api/v3alpha/cluster:pkg",
         "//envoy/api/v3alpha/core:pkg",
```

Comment:
Is the goal to eventually nuke the entire `envoy.api` in v3? I think this package namespace has lost its meaning over time, so would be excited to see that go away.